doc/fr/resources/memory.rst

```diff
@@ -1,6 +1,119 @@
 Mémoire
 =======
 
-`English <../en/02-mem.html>`_
+`English <../en/resources/memory.html>`_
 
-Matériel du sujet B...
+L'analyse des besoins en mémoire vive est similaire à celle pour les besoins en
+temps, excepté que les métriques sont différentes.
+
+Pour rappel, voici les deux principales options de l'ordonnanceur Slurm pour
+demander de la mémoire :
+
+- ``--mem=<quantité>``, quantité de mémoire
+  `par noeud de calcul <https://slurm.schedmd.com/sbatch.html#OPT_mem>`_.
+- ``--mem-per-cpu=<quantité>``, quantité de mémoire
+  `par coeur CPU <https://slurm.schedmd.com/sbatch.html#OPT_mem-per-cpu>`_.
+
+  - Par défaut : ``256M``
+
+Pourquoi se fixer une limite de mémoire?
+----------------------------------------
+
+- Dans les
+  `grappes nationales <https://docs.alliancecan.ca/wiki/National_systems/fr#Liste_des_grappes_de_calcul>`_,
+  la grande majorité des noeuds de calcul contiennent un peu moins de **4 Go
```

Comment:
J’utiliserais les mêmes unités que SLURM (M, G, T) plutôt que Mo, Go, To. Cela évite les erreurs dans les scripts de tâches. À strictement parler, il ne s’agit pas de Mo, Go, To, mais de MiO, GiO, TiO (multiples de 1024 vs 1000).